lib/phoenix_mtm/helpers.ex

```diff
@@ -50,6 +52,20 @@ defmodule PhoenixMTM.Helpers do
     )
   end
 
+  defp build_label_with_input(form, field, input_tag, label, label_opts, [nested: true]) do
+    [
+      label form, field, label_opts do
+        [{:safe, "#{label}"}, input_tag]
```

Comment:
I can't help but wonder if we should be calling `html_escape` here, instead of assuming it's a safely escaped string.

Did you have any thoughts on this when you wrote it?